lib/osml/tile_server/testing/ts_imagery.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2023 Amazon.com, Inc. or its affiliates.
+ */
+
+import { RemovalPolicy } from "aws-cdk-lib";
+import { IVpc } from "aws-cdk-lib/aws-ec2";
+import { BucketAccessControl } from "aws-cdk-lib/aws-s3";
+import {
+  BucketDeployment,
+  ServerSideEncryption,
+  Source
+} from "aws-cdk-lib/aws-s3-deployment";
+import { Construct } from "constructs";
+
+import { OSMLAccount } from "../../osml_account";
+import { OSMLBucket } from "../../osml_bucket";
+
+/**
+ * Configuration class for TSImagery Construct.
+ */
+export class TSImageryConfig {
+  /**
+   * Creates an instance of TSImageryConfig.
+   * @param {string} S3_IMAGE_BUCKET - The name of the S3 bucket where images will be stored.
+   * @param {string} S3_TEST_IMAGES_PATH - The local path to the test imager to deploy.
+   */
+  constructor(
+    public S3_IMAGE_BUCKET: string = "tile-server-test-images",
+    public S3_TEST_IMAGES_PATH: string = "assets/images/tile-server"
+  ) {}
+}
+
+/**
+ * Represents the properties for configuring the TSImagery Construct.
+ *
+ * @interface TSImageryProps
+ * @property {OSMLAccount} account - The OSML account to use.
+ * @property {IVpc} vpc - The Model Runner VPC configuration.
+ * @property {TSImageryConfig|undefined} [tsImageryConfig] - Optional configuration for TS Imagery.
+ * @property {string|undefined} [securityGroupId] - Optional security group ID to apply to the VPC config for SM endpoints.
+ */
+export interface TSImageryProps {
+  /**
+   * The OSML account to use.
+   *
+   * @type {OSMLAccount}
+   */
+  account: OSMLAccount;
+
+  /**
+   * The target vpc for the s3 bucket deployment.
+   *
+   * @type {IVpc}
+   */
+  vpc: IVpc;
+
+  /**
+   * Optional custom configuration for TSImagery.
+   *
+   * @type {TSImageryConfig|undefined}
+   */
+  tsImageryConfig?: TSImageryConfig;
```

Comment:
Make this ``config``

Reply:
After talking this over we'll have a consolidated effort to go through the CDK code base and update all these to config, but will merge this for now since it still matched the old pattern.